product_pricelist_direct_print_xlsx/wizards/product_pricelist_print.py

```diff
@@ -13,6 +14,32 @@ class ProductPricelistPrint(models.TransientModel):
 
     show_internal_category = fields.Boolean(string="Show internal categories")
 
+    print_child_categories = fields.Boolean(default=True, string="Print child categories")
+
+    def get_products_domain(self):
+        domain = super().get_products_domain()
+
+        if self.print_child_categories and self.categ_ids:
+            new_domain = []
+
+            for condition in domain:
+                if (
+                    isinstance(condition, tuple) and
+                    len(condition) == 3 and
+                    condition[0] == 'categ_id' and
+                    condition[1] == 'in'
```

Comment:
sorry, I dont understand these conditions

Reply:
find domain  [[("categ_id", "in", self.categ_ids.ids)]](https://github.com/chaule97/product-attribute/blob/0212ec692e79093ff6e6a1b92885f3fe1a2b222b/product_pricelist_direct_print/wizards/product_pricelist_print.py#L335) and replace it